src/js/utils/log.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-console */
-export const log: () => {} = (typeof console.log === 'function') ? console.log.bind(console) : noop;
+export const log: (msg: string) => {} = (typeof console.log === 'function') ? console.log.bind(console) : noop;
```

Comment:
We don't event use this in the player - only in some ad plugins. So I don't think you need to add the msg arg - (in reality this should be a rest arg since console.log can take any length of arguments.

```
interface Console {
    memory: any;
    assert(condition?: boolean, message?: string, ...data: any[]): void;
    clear(): void;
    count(label?: string): void;
    debug(message?: any, ...optionalParams: any[]): void;
    dir(value?: any, ...optionalParams: any[]): void;
    dirxml(value: any): void;
    error(message?: any, ...optionalParams: any[]): void;
    exception(message?: string, ...optionalParams: any[]): void;
    group(groupTitle?: string, ...optionalParams: any[]): void;
    groupCollapsed(groupTitle?: string, ...optionalParams: any[]): void;
    groupEnd(): void;
    info(message?: any, ...optionalParams: any[]): void;
    log(message?: any, ...optionalParams: any[]): void;
```

Reply:
i'll mark it optional for sure - was throwing errors without the arg